swift/ql/lib/codeql/swift/generated/Children.qll

```diff
@@ -0,0 +1,210 @@
+// generated by codegen/codegen.py
+import codeql.swift.elements
+
+cached
+Element getAChild(Element e) {
+  not exists(e.getResolveStep()) and
+  (
+    none()
+    or
```

Comment:
Is this `none()` or disjunction necessary?

Reply:
it makes it easier on the generation, as admittedly mustache is not very good at joining with a separator. 
In order to get rid of that I would need to specifically mark the first (or last) child property. Doable, but it would require more code (and tests). This way I can instead blindly append `or result = ...` for each child property, including the first.